services/ui-src/src/styles/index.scss

```diff
@@ -1,6 +1,7 @@
 // necessary to render the UsaBanner correctly
 $image-path: "~@cmsgov/design-system/dist/images";
 @use "@cmsgov/design-system/dist/css/index";
+@use '@cmsgov/design-system/dist/css/core-theme';
```

Comment:
Version 7 change. Now theres a core-theme to pull from with additional colors and variables!